test/unit/gateways/opp_test.rb

```diff
@@ -203,6 +226,14 @@ def failed_response(type, id, code='100.100.101')
     )
   end
 
+  def failed_store_response(id, code='100.100.101')
+    OppMockResponse.new(400,
+      JSON.generate({'id' => id, 'result' => {'code' => code, "des
+        cription" => 'invalid creditcard, bank account number or bank name'}, 'card' => {'bin' => '444444', 'last4Digits' => '4444', 'holder' => 'Longbob Longsen', 'expiryMonth' => '05', 'expiryYear' => '2018'},
```

Comment:
Looks like there's unintentional whitespace in `"des\n        cription"`

Reply:
@InPermutation thanks, fixed.